cli/tests/catalog_responses/hello_resolution.json

```diff
@@ -0,0 +1,303 @@
+[
+  [
+    {
+      "name": "toplevel",
+      "pages": [
+        {
+          "packages": [
+            {
+              "attr_path": "hello",
+              "broken": false,
+              "derivation": "???",
+              "description": "description",
+              "license": "license",
+              "locked_url": "github:NixOS/nixpkgs/ab5fd150146dcfe41fda501134e6503932cc8dfd",
+              "install_id": "hello",
```

Comment:
suggestion nonblocking: I think it may be good to use `hello_install_id` or something distinct from `name` to ensure we don't mix those up

Reply:
makes sense, fixing

Reply:
although that means we have to install it as `hello_install_id` as well 🤔